Week5/app/src/main/java/com/oliverperez/blockbusterchallenge/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.oliverperez.blockbusterchallenge.ui.login
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.oliverperez.blockbusterchallenge.model.LoginPrefs
+import com.oliverperez.blockbusterchallenge.R
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+        loginButton.setOnClickListener {
+            validateInput()
+        }
+        textFieldName.requestFocus()
+    }
+
+    private fun validateInput() {
```

Comment:
It would be much better if you moved validation logic outside of the Activity. You can create a helper function or a class to do it